File: src/my_project_typescript/easy_problems_typescript/from1to50/twoSum.ts
function twoSum(nums: number[], target: number): number[] {
    let temp: Array<number> = [];
    for (let i = 0; i < nums.length; i++){
        for (let j = i+1; j < nums.length; j++){
            if (nums[i] + nums[j] === target){
                temp.push(i)
                temp.push(j)
            }
        }
    }
    return temp
};

console.log(twoSum([2,7,11,15],9));
console.log(twoSum([3,2,4],6));
console.log(twoSum([3,3],6));

File: src/my_project_java/src/easy_problems/from_1_to_50/TwoSum.java
package easy_problems.from_1_to_50;

import java.util.HashMap;
import java.util.Map;

public class TwoSum {

    public int[] twoSum(int[] nums, int target){
        int[] result = new int[2];
        Map<Integer, Integer> map = new HashMap<Integer, Integer>();

        for ( int i = 0; i < nums.length; i++){

            if (map.containsKey())

            map.put(nums[i], i);
        }



        return result;

    }
}
